src/renderer/reader/components/Reader.tsx

```diff
@@ -736,8 +736,8 @@ class Reader extends React.Component<IProps, IState> {
         
         const isAudioBook = isAudiobookFn(this.props.r2Publication);
         const arrowDisabledNotEpub = isAudioBook || this.props.isPdf || this.props.isDivina;
-        const isFXL = this.isFixedLayout();
-        const isPaginated = this.props.readerConfig.paged;
+        // const isFXL = this.isFixedLayout();
```

Comment:
Oh, I didn't even realise that the previous/next arrows had been disabled for fixed-layout / pre-paginated publications! Good catch thanks